js/models/conversations.js

```diff
@@ -257,7 +267,7 @@
                 if (contact.isMe()) {
                     return [false, contact];
                 } else {
-                    return Promise.all([this.isUntrusted(), contact]);
+                    return Promise.all([contact.isUntrusted(), contact]);
```

Comment:
This is a separate bug from not handling missing contacts, right?

So the previous behavior was to treat each group member as untrusted if a single group member was untrusted, but now we correctly check per contact. 

Should this be `contact.safeIsUntrusted`?